### DISCUSSION TITLE
organization has been flagged.

### DISCUSSION THREAD

**Discussion by manamil-coder** [community/community#37245] [Other Feature Feedback, Questions, & Ideas] [answered]
How  i can this issue

**Comment by airtower-luna** [comment, marked as answer]
That's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help there. Good luck!

**Reply by manamil-coder** [reply]
The Snow-Dream-Studio organization has been flagged.
Please Fixed this error

**Reply by manamil-coder** [reply]
Why this organization has been flagged

**Reply by yalla-shoot-smart** [reply]
Hello my friend
Please look at my account also it has been flagged I don't know why
I opened a ticket and no response yet, her number is #1866989
